packages/google-pay-integration/src/google-pay-paypal-commerce/google-pay-paypal-commerce-script-loader.ts

```diff
@@ -164,6 +166,7 @@ export default class GooglePayPayPalCommerceScriptLoader {
             attributes: {
                 'data-partner-attribution-id': attributionId,
                 'data-client-token': clientToken,
+                ...(cspNonceExperiment && { 'data-csp-nonce': cspNonce }),
```

Comment:
It would be better to improve it that way

```
const isCSPNonceApplicable = cspNonceExperiment && cspNonce;

...(isCSPNonceApplicable && { 'data-csp-nonce': cspNonce })
```

To it for other places too pls

Reply:
fixed